webpack.prod.js

```diff
@@ -0,0 +1,62 @@
+const path = require("path");
+const HTMLWebpackPlugin = require("html-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const Dotenv = require('dotenv-webpack');
+
+module.exports = {
+  mode: "production",
+  entry: "./src/index.js",
+  output: {
+    path: path.join(__dirname, "/dist"),
+    filename: "bundle.js",
+  },
+
+  plugins: [
+    new MiniCssExtractPlugin({ filename: "App.css" }),
+    new CleanWebpackPlugin(),
+    new Dotenv({
+      path: './env.production',
+    }),
+  ],
+
+  optimization: {
+    minimizer: [
+      new TerserPlugin(),
+      new HTMLWebpackPlugin({
+        template: "./src/index.html",
+        minify: {
+          removeAttributeQuotes: true,
+          collapseWhitespace: true,
+          removeComments: true,
+        },
+      }),
+    ],
+  },
+  devServer: {
+    inline: false,
+    contentBase: "./dist",
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
```

Comment:
Use DRY principle for config also. Please move config duplications to webpack.base.js or smth like that

Reply:
Apply DRY Principle and saperation of concerns in project by creating saperate webpack.common.js file,